app/helpers/utils.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import logging.config
+import os
+from itertools import chain
+
+import yaml
+
+logger = logging.getLogger(__name__)
+
+
+def get_logging_cfg():
+    cfg_file = os.getenv('LOGGING_CFG', 'logging-cfg-local.yaml')
+
+    config = {}
+    with open(cfg_file, 'rt') as fd:
+        config = yaml.safe_load(fd.read())
+
+    logger.debug('Load logging configuration from file %s', cfg_file)
+    return config
+
+
+def init_logging():
+    config = get_logging_cfg()
+    logging.config.dictConfig(config)
+
+
+def get_registered_method(app, endpoint):
+    '''Returns the list of registered method for the given endpoint'''
+    all_methods = ['GET', 'HEAD', 'OPTIONS', 'POST', 'PUT', 'DELETE']
+    logger.debug(
+        'Url rules endpoints: %s', ','.join([r.endpoint for r in app.url_map.iter_rules()])
+    )
+    logger.debug('Request endpoint %s', endpoint)
+    return list(
+        chain.from_iterable([
+            r.methods if r.methods else all_methods for r in app.url_map.iter_rules(endpoint)
```

Comment:
As discussed, probably a small comment / link to the docu.
I stumbled over "why are all methods returned as allowed ones, when there is no r.methods at all for an endpoint?"

Reply:
Done